backend/prompt_studio/prompt_studio_core_v2/models.py

```diff
@@ -140,7 +140,7 @@ class CustomTool(DefaultOrganizationMixin, BaseModel):
     def delete(self, organization_id=None, *args, **kwargs):
         # Delete the documents associated with the tool
         if not check_feature_flag_status(FeatureFlag.REMOTE_FILE_STORAGE):
-            file_path = FileManagerHelper.handle_sub_directory_for_prompt_studio(
+            file_path = FileManagerHelper.handle_sub_directory_for_tenants(
```

Comment:
Was this a bug and we are reverting to the code that existed earlier?

Reply:
@gaya3-zipstack `handle_sub_directory_for_prompt_studio` is to be invoked only when feature flag is off. It should not have been added in the positive flow of the flag. Right now, this causes error while deleting prompt studio. Hence made this change.

Reply:
ok.